react/fields/FieldMessage/FieldMessage.js

```diff
@@ -19,7 +19,8 @@ export default class FieldMessage extends Component {
     valid: PropTypes.bool,
     message: React.PropTypes.oneOfType([
       React.PropTypes.string,
```

Comment:
You should be able to remove this line now, since a string is also a valid node.